docs/connectors/index.mdx

```diff
@@ -161,6 +161,33 @@ specified.
 See [Delta Lake output connector documentation](/connectors/sinks/delta)
 for an example of configuring the output buffer.
 
+## Fault tolerance
```

Comment:
We'll need documentation about how this is configured. Is it a property of the pipeline? Or configured per connector?

Reply:
It's a property of the pipeline, which automatically enables fault tolerance on all of the connectors.

I could not find documentation on how to configure a pipeline. That would be the right place to document this. For now, I added a note here:

```patch
diff --git a/docs/connectors/index.mdx b/docs/connectors/index.mdx
index d45e104b1..9a9a2f59c 100644
--- a/docs/connectors/index.mdx
+++ b/docs/connectors/index.mdx
@@ -166,7 +166,9 @@ for an example of configuring the output buffer.
 As a preview feature, Feldera supports fault tolerance, meaning that
 it can gracefully restart from the exact point of an abrupt shutdown
 or crash, picking up from where it left off without dropping or
-duplicating input or output.
+duplicating input or output.  To enable fault tolerance on a pipeline,
+set `"fault_tolerance": "latest_checkpoint"` in the pipeline
+configuration.
 
 When a fault-tolerant pipeline restarts, it loads its state from the
 most recent checkpoint, then it replays any data from input connectors
```